README.md

```diff
@@ -0,0 +1,11 @@
+# Template
+
+## Description
+
+Please provide consistent description of your repository. Explain what you develop and why it's better than alternatives (if there are any)
```

Comment:
It's better to just provide link to common wiki
Without any additional explanations
ause it's better to store info in one place

Reply:
I find it important providing _short_ description in readme & of course leave link to Wiki